src/filter.py

```diff
@@ -126,3 +127,68 @@ def release_type(results):
         for result in results
         if get_setting('include_release_' + result["release_type"], bool)
     ]
+
+
+def tv_season_episode(results, season, season_name, episode, global_ep, ep_year, season_year=0, start_year=0):
+    # Function for sorting large amount of not accurate torrents. Checks year season and episodes.
+    # The filtering shouldn't be strict. I'm trying not to lose suitable torrents.
+    filtered = []
+    for res in results:
+        name = res["name"].lower()
+        # Remove resolution
+        name = re.sub(r"\d+p", '', name)
+        log.debug(f"torrent: {name}")
+
+        if season_name and season_name in name:
+            filtered.append(res)
+            continue
+
+        year_pattern = r"(?P<from>(?:19|20)\d{2})(?:\s*-\s*(?P<to>[12]\d{3}))?"
```

Comment:
Do you have an example of something that has two years in there?

Reply:
![image](https://github.com/fugkco/script.elementum.jackett/assets/43271561/38cddec4-d6d5-43c9-9f4f-6ca009517c47)